pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/OpplysningerBruktIBeregningenEndretUttaksgrad.kt

```diff
@@ -0,0 +1,519 @@
+                )
+            }
+        }
+
+        showIf(alderspensjonVedVirk.regelverkType.isOneOf(AP2016, AP2025)) {
+            paragraph {
+                // vedleggBeregnTabellOverskrift_002
+                table(
+                    header = {
+                        column(columnSpan = 2, alignment = Element.OutlineContent.ParagraphContent.Table.ColumnAlignment.LEFT) {
+                            textExpr(
+                                Bokmal to "Opplysninger brukt i beregningen fra ".expr() + krav.virkDatoFom.format(),
+                                Nynorsk to "Opplysninger brukte i berekninga frå ".expr() + krav.virkDatoFom.format(),
+                                English to "Information used in the calculation as of ".expr() + krav.virkDatoFom.format()
+                            )
+                        }
+                        column(alignment = Element.OutlineContent.ParagraphContent.Table.ColumnAlignment.RIGHT) {}
+                    }
+                ) {
+                    showIf(trygdetidsdetaljerKap20VedVirk.anvendtTT.greaterThan(0)) {
+                        // vedleggTabellKap20Trygdetid_001
+                        row {
+                            cell {
+                                textExpr(
+                                    Bokmal to "Trygdetid".expr() + ifElse(alderspensjon.regelverkType.equalTo(AP2016), " etter kapittel 20", ""),
+                                    Nynorsk to "Trygdetid".expr() + ifElse(alderspensjon.regelverkType.equalTo(AP2016), " etter kapittel 20", ""),
+                                    English to "National insurance coverage".expr() + ifElse(alderspensjon.regelverkType.equalTo(AP2016), " pursuant to Chapter 20", "")
+                                )
+                            }
+                            cell {
+                                includePhrase(AntallAarText(trygdetidsdetaljerKap20VedVirk.anvendtTT))
+                            }
+                        }
+                    }
+
+                    showIf(oppfrisketOpptjeningVedVirk.opptjeningTilfortKap20.greaterThan(0)) {
+                        //  vedleggTabellKap20NyOpptjening_001
+                        row {
+                            cell {
+                                text(
+                                    Bokmal to "Ny opptjening",
+                                    Nynorsk to "Ny opptening",
+                                    English to "New accumulated pension capital"
+                                )
+                            }
+                            cell {
+                                includePhrase(KronerText(oppfrisketOpptjeningVedVirk.opptjeningTilfortKap20))
+                            }
+                        }
+                    }
+
+                    showIf(endretUttaksgradVedVirk.pensjonsbeholdning.greaterThan(0) and alderspensjonVedVirk.regelverkType.equalTo(AP2016)) {
```

Comment:
slå sammen disse to og inline tabellPensjonsbeholdning. Kan bare sjekke alderspensjonVedVirk.regelverkType.isOneOf(AP2016, AP2025)